tests/pass/shims/fs-with-isolation.rs

```diff
@@ -1,4 +1,5 @@
 //@ignore-target-windows: File handling is not implemented yet
+//@ignore-target-freebsd
```

Comment:
what goes wrong when this runs on freebsd? please add a FIXME

Reply:
no I just forgot to enable it again..